f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch

```diff
@@ -0,0 +1,484 @@
+diff --git a/Testing/script/travis_build.sh b/Testing/script/travis_build.sh
+index 32cc97f..69001c8 100755
+--- a/Testing/script/travis_build.sh
++++ b/Testing/script/travis_build.sh
+@@ -2,10 +2,10 @@
+ 
+ BMI_VERSION=master
+ CMAKE_VERSION_MAJOR=3.15
+-CMAKE_VERSION_MINOR=1
++CMAKE_VERSION_MINOR=3
+ MPI_VERSION=3.3.1
+ CCI_VERSION=2.1
+-OFI_VERSION=1.8.0
++OFI_VERSION=1.8.1
```

Comment:
Is this going to need a libfabric update also then?

Reply:
We are already past that version in cart itself, my local cart master shows as:
fi_info: 1.9.0a1

Reply:
But is the RPM there yet?

Reply:
Ahhh.  Seems libfabric RPM is current to CaRT:
https://github.com/daos-stack/libfabric/blob/a2b3e756c048f53a853973098d6944555e0784f3/libfabric.spec#L16